crates/q_cli/src/cli/chat/mod.rs

```diff
@@ -161,6 +162,7 @@ const WELCOME_TEXT: &str = color_print::cstr! {"
 <em>/quit</em>         <black!>Quit the application</black!>
 
 <cyan!>Use Ctrl(^) + j to provide multi-line prompts.</cyan!>
+<cyan!>Use Ctrl(^) + k to fuzzily search commands and context (use tab to select multiple files).</cyan!>
```

Comment:
This is overriding the existing rustyline (and every other shell environment) binding for ctrl-K of kill to end of line. I think another binding is needed.